fields/selected.go

```diff
@@ -0,0 +1,18 @@
+package fields
+
+import (
+	"context"
+
+	"github.com/graph-gophers/graphql-go/exec"
+	"github.com/graph-gophers/graphql-go/types"
+)
+
+// SelectedField retrieves the selected field passed via the context during the request execution
+func SelectedField(ctx context.Context) *types.SelectedField {
```

Comment:
kinda stuttery?

Reply:
I renamed it to `Get` and `GetRoot`, let's see what you think of it 
The name of the package is `fields` so `fields.Get(ctx)` and `fields.GetRoot(ctx)` both returning `*types.SelectedFields` so it makes sense to me